.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,9 +15,10 @@ A clear and concise description of what the bug is.
 
 Select the method used to integrate with reCAPTCHA Mobile.
 
-* [ ] SPM
-* [ ] Cocoapods
-* [ ] Direct Download
+* [ ] SPM (iOS)
+* [ ] Cocoapods (iOS)
+* [ ] Direct Download (iOS)
+* [ ] GMaven (Android)
```

Comment:
What do you think about having a different template for Android? We can have multiple custom template so when someone is creating the bug they will chose between Android and iOS. 

And we could also use issue forms that would make it easier for them to fill an Issue 

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

What do you think?

Reply:
I think they are similar enough for now, we can revisit as they diverge.